pkg/osd_cluster_upgrader/upgrader.go

```diff
@@ -372,22 +376,21 @@ func performUpgradeVerification(c client.Client, logger logr.Logger) (bool, erro
 	readyRs := 0
 	totalRs := 0
 	for _, replica := range replicaSetList.Items {
```

Comment:
out of topic, but the `replica` here should be `replicaSet` or `rs`? to be more accurate

Reply:
Thanks - have now renamed them better in commit ada6a4d